### ISSUE TITLE
Rethink of "most popular" sorting of public groups/topics

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#101]
Copying mine and @ilmartyrk's Slack conversation here about the "most popular" public groups and topics currently being filtered more like "most recent", for consideration at some point:  

![image](https://user-images.githubusercontent.com/61320526/228477545-e0ef9fb4-caec-471f-af04-f56bb4c413c8.png)

**Comment by anettlinno** [comment]
Thanks for bringing this up, @BeccaMelhuish :) 

If we are talking about "Popular" groups and topics, then we should become on the same page. According to Oxford Dictionary, adjective "**popular**" has many meanings, but I would bring out following two meanings: "Liked or admired by many people or by a particular person or group" but also "Prevalent among the general public". 

This means that if we are talking about popular groups and topics, then we should either publish topics, which are commonly liked (upvoted) or which have the most actions BY USERS (general public). I personally lean to the second definition -- **prevalent among the general public** because in this case our system could highlight groups and topics which have most actions by users who participate in discussion and decision-making process (excluding admin users actions like changing the topic text etc). Including most topics (in a group) and arguments and replies, likes and dislikes -- activity measured in the argumentary. 

If currently we display latest updated groups under the "Most popular" then this should be changed (at least copy). I think that on the front page it is more valuable to display groups which are active (latest updated), because if there are groups which were super popular and crowded 3 years ago, but the good old "crises" passed, our system would still show it among most popular groups and topics. This is dangerous, because Users might think that the platform is outdated or is not active any more (we have had these questions from our Users after they have read very old topics on our platform´s front page). 

So, long story short. I would leave most active groups and topics on the front page and change the copy accordingly. @BeccaMelhuish @ilmartyrk your thoughts are very much appreciated!

PS!

In redesign Users can sort out groups according to several indicators, incl. order: 
<img width="525" alt="Popular groups" src="https://github.com/citizenos/citizenos-fe/assets/45175679/5fe43a4b-0200-442f-a417-0d25fd96867e">

**Comment by anettlinno** [comment]
Haa! And I just checked that in redesign we use the same language (copy), so it should be changed there as well: 

<img width="522" alt="Most popular" src="https://github.com/citizenos/citizenos-fe/assets/45175679/4cecef9b-62b6-4270-bc0f-b6e8a60a8bc6">

**Comment by anettlinno** [comment]
Triage 85. Please help us to find good copy for "Latest updated topic" texts @oksks @ssin1901.

**Comment by BeccaMelhuish** [comment]
This was changed to "most recent public groups" in the text review, so will close :)
